lib/PrintAsClang/ClangSyntaxPrinter.h

```diff
@@ -135,6 +138,13 @@ class ClangSyntaxPrinter {
                       llvm::function_ref<void(raw_ostream &OS)> bodyPrinter,
                       NamespaceTrivia trivia = NamespaceTrivia::None) const;
 
+  /// Prints the C++ namespaces of the outer types for a nested type.
+  /// E.g., for struct Foo { struct Bar {...} } it will print
+  /// namespace FooNested { ..body.. } // namespace FooNested
```

Comment:
Minor: I think this name is now underscored, right?

Reply:
Whoops, indeed!